grails-databinding/src/main/groovy/org/grails/databinding/converters/DateConversionHelper.groovy

```diff
@@ -35,6 +35,9 @@ class DateConversionHelper implements ValueConverter {
     List<String> formatStrings = ['yyyy-MM-dd HH:mm:ss.S',"yyyy-MM-dd'T'HH:mm:ss'Z'","yyyy-MM-dd HH:mm:ss.S z"]
 
     Object convert(value) {
+        if (''.equals(value.toString().trim())) {
+            return null
+        }
```

Comment:
So this makes the code no longer `null` safe which could be an issue. Also it would be better if it used `GrailsStringUtils.isBlank(value)`